### PR TITLE
Store maven-indexer data on PV

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
This PR won't do the trick on its own. https://github.com/fabric8-analytics/fabric8-analytics-jobs/pull/214 needs to be merged as well.

File: f8a_jobs/handlers/maven_popular_analyses.py [hunk 1]
```diff
@@ -125,7 +126,15 @@ def _top_tags_projects(self):
 
     def _use_maven_index_checker(self):
         maven_index_checker_dir = os.getenv('MAVEN_INDEX_CHECKER_PATH')
-        target_dir = os.path.join(maven_index_checker_dir, 'target')
+        target_dir = '/pv/index-checker'
+        try:
+            os.makedirs(target_dir, exist_ok=True)
+        except OSError as exc:
+            if exc.errno == errno.EEXIST and os.path.isdir(target_dir):
```
Comment (jpopelka) [inline f8a_jobs/handlers/maven_popular_analyses.py]: Is it needed when we use `exist_ok=True` ?
https://docs.python.org/3/library/os.html#os.makedirs

File: f8a_jobs/handlers/maven_popular_analyses.py [hunk 2]
```diff
@@ -125,7 +126,15 @@ def _top_tags_projects(self):
 
     def _use_maven_index_checker(self):
         maven_index_checker_dir = os.getenv('MAVEN_INDEX_CHECKER_PATH')
-        target_dir = os.path.join(maven_index_checker_dir, 'target')
+        target_dir = '/pv/index-checker'
```
Comment (jpopelka) [inline f8a_jobs/handlers/maven_popular_analyses.py]: - shouldn't it end with `target` ?
- maybe set something like `ENV MAVEN_INDEX_CHECKER_TARGET_DIR='/pv/index-checker'` in `Dockerfile` and read it here ?

**Comment by msrb** [comment]
@jpopelka updated, PTAL.

**Comment by msrb** [comment]
omg, clicked merge myself by accident. @jpopelka please let me know if you want me to change/fix anything else

**Comment by jpopelka** [comment]
Let's Get This Money

**Pull request merged**